src/hyp3_isce2/process.py

```diff
@@ -47,6 +52,11 @@ def topsapp_burst(
     insar_roi = get_region_of_interest(ref_metadata.footprint, sec_metadata.footprint, is_ascending=is_ascending)
     dem_roi = ref_metadata.footprint.intersection(sec_metadata.footprint).bounds
     print(insar_roi, dem_roi)
+
+    download_aux_cal(aux_cal_dir)
+    for granule in (ref_params.granule, sec_params.granule):
+        orbit_file, _ = downloadSentinelOrbitFile(granule, orbit_dir)
```

Comment:
`orbit_file` is never used, can we just call the function without capturing its return value?

Reply:
Makes sense, I'll remove it.